Builds/VisualStudio2015/README.md

```diff
@@ -33,6 +29,9 @@ page, run the installer, and follow the directions.
 The **Visual Studio 2015 Community** edition is available for free, while paid editions
 may be used for an free initial trial period.
 
+Any version of Visual Studio 2015 may be used, including the Visual Studio Community
+Edition which is available under a free license.
+
```

Comment:
The C++ compiler is no longer installed by default in visual studio. We should remind people to choose "Custom" installation, and under "Programming Languages" select "Visual C++"